tests/integration/devfile/utils/utils.go

```diff
@@ -521,31 +521,34 @@ func OdoWatchWithIgnore(odoV2Watch OdoV2Watch, context, flag string) {
 }
 
 func validateContainerExecListDir(odoV1Watch OdoV1Watch, odoV2Watch OdoV2Watch, runner interface{}, platform, project string, isDevfileTest bool) error {
-	var stdOut string
-
+	var folderToCheck, podName string
+	cliRunner := runner.(helper.CliRunner)
 	switch platform {
 	case "kube":
 		if isDevfileTest {
-			folderToCheck := "/projects"
+			folderToCheck = "/projects"
 			if odoV2Watch.FolderToCheck != "" {
 				folderToCheck = odoV2Watch.FolderToCheck
 			}
 			cliRunner := runner.(helper.CliRunner)
```

Comment:
I think you missed removing `cliRunner` from here, no?

Reply:
yes updating

Reply:
Updated